gcalbot/gcalbot/html.go

```diff
@@ -325,29 +423,50 @@ const tmplConfig = `{{template "header" .}}
 		<div class="daily-schedule">
 		<div class="row">
 		<label for="ds_days">Daily schedule messages should be sent... </label>
-		<select name="ds_days">
-			{{range .DSDaysOptions}}
-				<option value="{{.Days}}" {{if eq .Days $.DSDays}} selected {{end}}>{{.Title}}</option>
-			{{end}}
-		</select>
+		<div class="select-container">
+			<select name="ds_days">
+				{{range .DSDaysOptions}}
+					<option value="{{.Days}}" {{if eq .Days $.DSDays}} selected {{end}}>{{.Title}}</option>
+				{{end}}
+			</select>
+			<div class="caret">
+				<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
+					<path fill-rule="evenodd" d="M.64 2h6.72c.354 0 .64.265.64.592a.567.567 0 0 1-.162.394l-3.36 3.486a.677.677 0 0 1-.956 0L.162 2.986a.561.561 0 0 1 .052-.836A.67.67 0 0 1 .64 2z"/>
+				</svg>
+			</div>
+		</div>
 		</div>
 
 		<div class="row">
 		<label for="ds_schedule">Send reminders for events... </label>
-		<select name="ds_schedule">
-			{{range .DSScheduleOptions}}
-				<option value="{{.Schedule}}" {{if eq .Schedule $.DSSchedule}} selected {{end}}>{{.Title}}</option>
-			{{end}}
-		</select>
+		<div class="select-container">
+			<select name="ds_schedule">
+				{{range .DSScheduleOptions}}
+					<option value="{{.Schedule}}" {{if eq .Schedule $.DSSchedule}} selected {{end}}>{{.Title}}</option>
+				{{end}}
+			</select>
+			<div class="caret">
+				<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
+					<path fill-rule="evenodd" d="M.64 2h6.72c.354 0 .64.265.64.592a.567.567 0 0 1-.162.394l-3.36 3.486a.677.677 0 0 1-.956 0L.162 2.986a.561.561 0 0 1 .052-.836A.67.67 0 0 1 .64 2z"/>
+				</svg>
+			</div>
+		</div>
 		</div>
 
 		<div class="row">
 		<label for="ds_time">Daily schedule should be sent at... </label>
-		<select name="ds_time">
-			{{range .DSTimeOptions}}
-				<option value="{{.Minute}}" {{if eq .Minute $.DSTime}} selected {{end}}>{{.Title}}</option>
-			{{end}}
-		</select>
+		<div class="select-container">
+			<select name="ds_time">
+				{{range .DSTimeOptions}}
+					<option value="{{.Minute}}" {{if eq .Minute $.DSTime}} selected {{end}}>{{.Title}}</option>
+				{{end}}
+			</select>
+			<div class="caret">
+				<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
```

Comment:
would be nice to not have to hardcode the same svg a bunch of times - maybe put this in a variable or with the other image?